server/views/pages/contact-us.ejs

```diff
@@ -0,0 +1,32 @@
+<% include ../helpers/helpers %>
+
+<!DOCTYPE html>
+<html lang="en" class="govuk-template">
+  <head>
+    <% include ../partials/head %>
+  </head>
+  <body class="govuk-template__body">
+    <% include ../partials/header %>
+
+    <div class="govuk-width-container">
+      <% include ../partials/beta-banner %>
+      <a href="javascript:history.back()" class="govuk-back-link">Back</a>
+      <main id="main-content" role="main" class="govuk-main-wrapper">
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l">Contact us</h1>
+            <p class="govuk-body">
+              For queries about Check My Diary please call the Service Desk on
+              <span class="govuk-body" style="white-space: nowrap;">0203 788 4636</span>.
+            </p>
+            <p class="govuk-body">
+              Alternatively, you can <a href="mailto:checkmydiary@digital.justice.gov.uk"  class="govuk-link">email us</a>.
```

Comment:
Is it worth adding the email to the copy? I don't have an email app setup on my computer, so `mailto:` is always a pain on desktops/laptops.

Reply:
I agree it should be there as well. I think this possibly highlights the immaturity of the ticket though. It has already come through the correct channels (UX, content and product).

Reply:
> highlights the immaturity of the ticket

😨